File: CS50 Labs and Problem Sets/Mario.c
#include <cs50.h>
#include <stdio.h>


int main (void)
{
    // getting a positive interger from user
    int height;
    int width;
    int dots;
    width=0;
    do
    {
        height = get_int("Height: ");
    }
    while (height<1 || height>8);
    
    dots = height - 1;
    
    for (int i=0; i<height; i++)
    {
        for (int z=0; z<dots; z++)
        {
            printf(".");
        }
        
        for (int j=0; j<width; j++)
        {
            printf("#");
        }
        width++;
        dots--;
        printf("#");
        printf("\n");
    }
}

File: CS50 Labs and Problem Sets/Python/mario.py
height = input("Height: ")
height = int(height)
while height < 1 or height > 8:
    height = input("Height: ")
    height = int(height)
    
dots = height - 1
width = 1

for i in range(height):
    for j in range(dots):
        print("." , end = '')
        
    for k in range(width):
        print("#" , end = '')
        
    dots -= 1
    width += 1
    print()

    
